base/src/main/res/values/strings.xml

```diff
@@ -20,4 +20,7 @@
     <string name="credits">Credits</string>
     <string name="name_of_this_language">English</string>
     <string name="tour_argument_key" translatable="false">tour</string>
+    <string name="leaveTour">Leave Tour</string>
+    <string name="stay">STAY</string>
```

Comment:
Please use camelcase for now, we should apply the all-caps attribute at the theme level.